### PR TITLE
Added message id functionality. top kek

### PR DESCRIPTION
**Pull request by KyeNormanGill**
Haven't tested it. Surprise surprise. Probably doesn't work. Who knows.

P.S. Could probably be better if i make a regex to check if it's a valid snowflake thingy

### CODE REVIEW

File: modules/translate/main.js [hunk 1]
```diff
@@ -78,6 +78,11 @@ module.exports = (bot, channel, user, args, id, message, extra) => {
         return channel.fetchMessages({ before: id, limit: 1 })
             .then(msg => translate(msg.first().content))
             .catch(error => util.error(error, "translate", channel));
+    }
+    else if (isNaN(query)){
```
Comment (kurisubrooks) [inline modules/translate/main.js]: collapse this `else if` onto the previous line, with a space trailing the closing curly bracket

Comment (kurisubrooks) [inline modules/translate/main.js]: also, shouldn't you be checking for `!isNaN(query)`? why not just `Number(query)`?

Comment (KyeNormanGill) [inline modules/translate/main.js]: Wooooopsy

Comment (kurisubrooks) [inline modules/translate/main.js]: also, add a space after the condition and before the opening curly bracket. lol

**Comment by KyeNormanGill** [comment]
I pushed the fix idk if it added to this PR. I'm not the best at gitub lul

File: modules/translate/main.js [hunk 2]
```diff
@@ -78,7 +78,7 @@ module.exports = (bot, channel, user, args, id, message, extra) => {
         return channel.fetchMessages({ before: id, limit: 1 })
             .then(msg => translate(msg.first().content))
             .catch(error => util.error(error, "translate", channel));
-    } else if (isNaN(query)){
+    } else if (!isNaN(query)){
```
Comment (iCrawl) [inline modules/translate/main.js]: Still missing the space after the statement.

**Pull request merged**